test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/VideoViewTest.cs

```diff
@@ -27,16 +27,17 @@ public myPlayer(IntPtr p) : base(p, null)
         myPlayer player;
         string resourcePath;
         const string tag = "NUITEST";
+        View dummy;
         public void Activate()
         {
             win = NUIApplication.GetDefaultWindow();
-            win.BackgroundColor = Color.Green;
+            win.BackgroundColor = new Color(1, 1, 1, 0);
```

Comment:
What's the purpose of this example?

Reply:
Hello, This is related to this JIRA : https://code.sec.samsung.net/jira/browse/GRE-1804
SafeNativePlayerHandle() was added to get native mm-player handle from VideoView and this sample is to test it. Thank you.